source/Halibut.Tests/Support/TestPollingClient.cs

```diff
@@ -0,0 +1,19 @@
+using System;
+
+namespace Halibut.Tests.Support
+{
+    public class TestPollingClient : ITestPollingClient
+    {
+        public int ListeningPort { get; }
+        public string WebSocketPath { get; }
+
+        TestPollingClient(int listeningPort, string webSocketPath)
```

Comment:
nit: Is this really PollingClientDetails or similar rather than a client that we think of in terms of client and service

Reply:
After discussion, we decided to simplify it down to just a URI